### ISSUE TITLE
Ebligi la uzon de learn.esperanto.com kiel Progressive Web App

### ISSUE THREAD

**Issue by LaPingvino** [Esperanto/kurso-zagreba-metodo#117]
[La necesaj dosieroj, kreitaj per pwabuilder.com](https://github.com/Esperanto/kurso-zagreba-metodo/files/2333878/learn.Esperanto.com-PWA-web.zip)

La ideo de ĉi tio estas ke oni povas instali la retejon loke kie tio estas subtenata, tio estas en Windows 10, ĉie kie funkcias Chrome, poŝtelefonoj, kun la dosieroj eksterrete, bela emblemo ktp.

**Comment by LaPingvino** [comment]
Funkcianta versio nun estas en https://esperanto.github.io/kurso-zagreba-metodo/, ankoraŭ necesas aŭtomatigi la fingeneradon de ĉi tio.

**Comment by LaPingvino** [comment]
Generado aŭtomatigita, restas eventuale apliki la ŝanĝojn ankaŭ al https://learn.esperanto.com, al kiu mi ne havas aliron.

**Comment by georgjaehnig** [comment]
Kara Joop, tio estas bona idea, havi la retejon kiel PWA.

Mi ĵus provis kaj instalis ĝin al mia saĝtelefono (MIUI Android 7), tio funkciis, ankaŭ mi povis uzi la retejon ene de la apo. Sed kiam mi malŝaltis la reton en mia telefono, la apo ne plu funkciis.

Mi kredas ke la intenco estas ke ĝi plu funkcias, ĉefe kiam oni ne havas reton, ĉu ne? Ĉu vi povas ripari tion?

Kaj ankaŭ: Mi agordis ke la retejon sur https://esperanto.github.io/kurso-zagreba-metodo/  plusendas al https://learn.esperanto.com, por ne havi duoblan enhavon. Ĉu vi ŝanĝis tion?

**Comment by Jxajroad** [comment]
Por mi tiu estas greka!

Em sex, 31 de ago de 2018 às 06:33, Georg Jähnig <notifications@github.com>
escreveu:

**Comment by LaPingvino** [comment]
Mi metis la aplikaĵon en esperanto.github.io/kurso-zagreba-metodo, ĉar tie antaŭe nur estis index.html kun eta teksto kiu faris nenion (ne redirektis).

**Comment by LaPingvino** [comment]
Mi mem ne havas aliron al learn.esperanto.com, do tie nenio estas ŝanĝita, mi esperas ke vi povu ŝanĝi tion.

**Comment by georgjaehnig** [comment]
> tie antaŭe nur estis index.html kun eta teksto kiu faris nenion (ne redirektis).

Nu, jes, ĝi alidirektis de https://esperanto.github.io/kurso-zagreba-metodo/html/output/ per `<meta>`, vidu jen: b7f486c890e88c7e70049755ee1010530fa6d5d4

Tio estis la antaŭe uzata retejo. Kelkaj ligiloj jam ligis (kaj ligas) tien, do mi volas ke tiuj lernantoj estas bone alidirektitaj al la nuna, ĝusta https://learn.esperanto.com/ 

Tial ni ne povas uzi tiun retejon por testi.

Mi nun reagordis la antaŭan retejon.

> Mi mem ne havas aliron al learn.esperanto.com, do tie nenio estas ŝanĝita, mi esperas ke vi povu ŝanĝi tion.

Mi ege, ege volonte uzos la kodon de la PWA! Mi mem jam ŝatis krei tion.

Nur en la nuna stato ŝajnas, ke la PWA ne bone funkcias: Ĝi ja ne ebligas la uzanton uzi la retejon senrete, ĉu?

Do bonvole ŝanĝu la kodon ke ĝi funkcias, ke ĝi elŝutas (kaj poste daŭre ĝisdatigas la tutan retejon). Ni ankaŭ ĉi tie povos babili pri tio. Eble ankaŭ kreu novan branĉon por ĝi.

Certe vi bezonos testan servilon por tio. Ĉu vi havas iun?

**Comment by LaPingvino** [comment]
Ĝi ja ebligas tion, post unuafoja vizitado de la bezonataj paĝoj. Mi kreis la ServiceWorker per pwabuilder.com, vi povus vidi tie pri alia stilo elŝuti la paĝon se vi volas.
Testante, ĝi NE redirektas, ankaŭ ne per meta, do mi ne konsentas pri la nepovo testi. La gh-pages esence restis neuzata praktike.
